safe/reportv4/impact_report.py

```diff
@@ -330,15 +345,57 @@ def output_folder(self, value):
             os.makedirs(self._output_folder)
 
     @property
-    def impact_layer(self):
+    def impact_function(self):
+        """Getter for impact function instance to use
+
+        :rtype: safe.impact_function_v4.impact_function.ImpactFunction
+        """
+        return self._impact_function
+
+    @property
+    def hazard(self):
+        """Getter to hazard layer
+
+        :rtype: qgis.core.QgsVectorLayer
+        """
+        return self._hazard_layer
```

Comment:
Sorry, I meant to call `self._hazard` in the object too. We win 6 characters for readability ;-)